common/scala/transformEnvironment.sh

```diff
@@ -21,6 +21,18 @@ props=()
 for var in $configVariables
 do
     value=$(printenv "$var")
+    #allow us to derefence environment variables, e.g. CONFIG_some_key=$SOME_ENV_VAR
+    if [[ $value == \$* ]]
+    then
+        varname=${value:1}
+        value2=${!varname}
```

Comment:
- comment `# '!' dereferences the variable`
- rename `value2` to `dereferencedValue`?